pandas/core/ops/array_ops.py

```diff
@@ -217,7 +227,7 @@ def arithmetic_op(
 
 
 def comparison_op(
-    left: Union[np.ndarray, ABCExtensionArray], right: Any, op
+    left: Union[np.ndarray, ABCExtensionArray], right: Any, op, str_rep, eval_kwargs
```

Comment:
``str_rep`` and ``eval_kwargs`` don't seem to be used here, is that right? Could they be removed, or have default arguments of ``None``?

Reply:
Also, if they need to stay, could you add type hints in order to ease understanding? I assume they have types ``Optional[str]`` and ``dict``, respectively?